apps/frontend/src/libs/components/table/mock-data.ts

```diff
@@ -0,0 +1,62 @@
+import { type ColumnDef } from "@tanstack/react-table";
+
+type Person = {
+	createdAt: string;
+	name: string;
+	permissions: string;
+	projects: string;
+};
+
+const data: Person[] = [
```

Comment:
we do not need any mock data in the PRs

Reply:
The task says that in order to be able to test, you need to add this component to the main page with mocked data. Should I remove mocked data?

Reply:
i am sorry, i did not notice that comment in the task. then i will resolve the comment 🙏